microsetta_interface/templates/update_address.jinja2

```diff
@@ -201,6 +229,13 @@ $(document).ready(function(){
                     <div class="form-group">
                         <label for="postal" name="postal_label">{{ _('Postal Code') }}*</label>
                         <input id="postal" name="postal" class="form-control" type="text" value="{{ user_info['postal_code'] |e }}" />
+                    </div>
+                     <div class="form-group" id="form_group_address_type">
+                            <label for="address_type" name="address_type_label">{{ _('Address Type') }}</label>
+                            <select id=address_type name=address_type class="form-control">
```

Comment:
Please consistently use quotes around HTML tag parameters, e.g. <select id="address_type" not <select id=address type . There are several instances throughout the PR, please check through the code to make sure they're consistent.